toolkit/tools/imagegen/attendedinstaller/views/installerview/installerview.go

```diff
@@ -147,6 +147,11 @@ func (iv *InstallerView) Primitive() tview.Primitive {
 
 // OnShow gets called when the view is shown to the user
 func (iv *InstallerView) OnShow() {
+	err := speakuputils.StartSpeakup()
```

Comment:
No need to stop speakup?

Reply:
Nope, the purpose of this is to ensure that if we choose the speechless installer, we can re-enable TTS by navigating back to this selection screen. If speakup is already started, then attempting to start the service will have no effect.